### ISSUE TITLE
offline routes marked as "local"

### ISSUE THREAD

**Issue by monikakrawczak** [CodeForPoznan/pah-fm#148]
A user can save a route offline in local storage but it isn't marked as "local".

## current behavior
offline routes are not marked as ''local"

## expected behavior
offline routes are marked as ''local"

### sidenote
That could be a reason why offline routes are not synced to backend

(split from #40)

**Comment by adamgajzlerowicz** [comment]
Is the sync not working? Are you getting any errors?
Also - what do you mean by marking routes as local?

**Comment by magul** [comment]
I re-check that functionality and eventually, drives (routes) are synced with the backend (but it's not triggered by switching to online, it's triggered by re-routing to routes view, so You need to switch to route list twice to see sync in action).

**Comment by magul** [comment]
Also, this is only a split functionality from #40 where @jacol12345 requested such remark.

I don't know if it will be used anywhere (obviously application right now recognize which drive is synced already and which one is not). I assume that from @jacol12345 it would be an attribute `"local": true` set on drive object.

Of course feel free to discuss if we need it at all.

**Comment by adamgajzlerowicz** [comment]
I am still not sure where you are getting the `"local": true` thing. I can't see see it in that story also I don't think that was a requrement of any sort.

Let me elaborate on how the sync works:

The frontend app currently has 2 separate containes for data in local storage. 
One is for routes that are in local storage only, and the other one for the routes that come from api (were previously synced)

You can add as many drives in the app, but at some point you need to dispatch the `SYNC` command. 

When sync is triggered, the app will send routes from the store slice to the api one by one (as previously discussed). Once the sync is finished, state from api will be re-fetched.

Now - sync command will be dispatched to store in one of three cases: 
- app is refreshed (loaded)
- drive is added
- internet connection is detected

So there is a lot of moving parts here, thus the reason for asking if you are getting any errors etc. Also - which browser are you using?

I did test the feature just last night on Chrome OSx and on ubuntu Chrome - it worked perfect. Perhaps there is an edge case we are not covering? 

Please provide debug info :)

**Comment by magul** [comment]
In #40 issue description was edited (last time by me).

The previous value there was:
> jacol12345 edited on Nov 7, 2018
> If user saves a route offline it is saved in local storage and marked as "local".
> Creating a route offline should display a message "you created a route - it will be synced once you are online".
> Once user is logged and on-line the routes they created while offline are synced with backend.

We just (with @monikakrawczak ) split this requirement to separate issue (here). I accept the architecture that You described above and agree that we do not need this `local` mark on drive/routes. If that's also ok with @jacol12345 we can close this issue right away.

Regarding the sync issue, I'm using Ubuntu with Google Chrome. I will show You it on Wednesday.

**Comment by jacol12345** [comment]
Please do not wait for me with decisions - I need to prep demo and accounts for Wednesday, I will (most likely) not have enough time to read&comment on this discussion.

**Comment by magul** [comment]
So per @adamgajzlerowicz description how architecture works here, I'm closing it now.
